actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -172,6 +172,25 @@ def _filtered_referrer # :nodoc:
       #   link_to "External link", "http://www.rubyonrails.org/", target: "_blank", rel: "nofollow"
       #   # => <a href="http://www.rubyonrails.org/" target="_blank" rel="nofollow">External link</a>
       #
+      # ==== Turbo
+      #
+      # Rails 7 ships with Turbo enabled by default. The <tt>data-turbo-method</tt> HTML attribute can be used
+      # to perform a Turbo link visit with a different HTTP method.
+      #
+      # A common use case is to insert links to perform update or delete operations (i.e. non-<tt>GET</tt> requests) in a form.
+      # For example, you might use +link_to+ in an edit form to delete a record:
+      #
+      #   link_to "Delete profile", @profile, data: { turbo_method: :delete }
```

Comment:
This example should probably be a `button_to`. Is there a more appropriate example we can use here?

Reply:
You're right, @ghiculescu ! I've added an expanded example of `button_to` that fails, and a short explanation on why `link_to` with `data-turbo-method` should be used instead in this commit: https://github.com/rails/rails/pull/47505/commits/1e568cd47103fd7ce8aa1f3fbe16779a9e73e23f